layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/SiteNavigationRestService.java

```diff
@@ -158,6 +158,38 @@ public Response undoDeleteNode(@Context
       return Response.serverError().build();
     }
   }
+  @PUT
```

Comment:
```suggestion
  @PATCH
```
since we will not update the entire entity